### DISCUSSION TITLE
Dependent for Diesel CLI

### DISCUSSION THREAD

**Discussion by DaoHuyTuan** [diesel-rs/diesel#4167] [Q&A]
Hi, i config diesel and axum, very confuse when i need to install database i want to use which diesel need some lib of the database for migration if not it will have some error when cargo run and cargo build 
```
Link clang: error: linker command failed with exit code 1
```
or some error about link CC and GCC 

what i try to do is using db run on docker on something similar, other language separate and make lib not depend too much on env of machine

**Comment by weiznich** [comment]
Please see the "Installing Diesel CLI" section of the [Getting Started](http://diesel.rs/guides/getting-started.html) guide for an answer. Especially the box "A note on installing diesel-cli" is relevant, although you likely to read the whole section.
